molgenis-core-ui/src/main/resources/js/jquery.molgenis.attributemetadata.table.js

```diff
@@ -34,7 +34,8 @@
             });
         }
 
-        if (attributeMetadata.fieldType === 'CATEGORICAL') {
+        var type = attributeMetadata.fieldType
+        if (type === 'CATEGORICAL' || type === 'CATEGORICAL_MREF' || type === 'MREF' || type === 'XREF') {
```

Comment:
CATEGORICAL_MREF is correct, but MREF and XREF shouldn't be here since the number of referred entities can be very large (also see my comment in issue #5839)